src/otx/core/data/dataset/tile.py

```diff
@@ -74,6 +74,8 @@ def __init__(
         overlap: tuple[float, float],
         threshold_drop_ann: float,
     ) -> None:
+        # NOTE: clip overlap to [0, 0.9]
+        overlap = max(0, min(overlap[0], 0.9)), max(0, min(overlap[1], 0.9))
```

Comment:
Should we document the overlap range and clip behavior in the class description?

Reply:
Thanks Vlad, I updated the class docstring.